org-code-javabuilder/theater/src/test/java/org/code/theater/StageTest.java

```diff
@@ -61,7 +64,7 @@ public void tearDown() {
   void drawLineDefaultsToBlack() {
     s.removeStrokeColor();
     s.drawLine(0, 0, 100, 100);
-    verify(graphics).setColor(Color.BLACK);
+    verify(graphics).setColor(java.awt.Color.BLACK);
```

Comment:
Why did this need to change to java.awt.Color?

Reply:
similarly to above, we now need both `java.awt.Color` and `org.code.media.Color`. I made `org.code.media.Color` the default, so we have to explicitly reference `java.awt.Color` if we want to use it.

Reply:
👍